src/Themes/Navigation.ios.js

```diff
@@ -0,0 +1,49 @@
+const tabBarBackgroundColor = '#003a66'
+const navBarButtonColor = '#ffffff'
+const tabBarButtonColor = '#ffffff'
+const navBarTextColor = '#ffffff'
+const tabBarSelectedButtonColor = '#ff505c'
+const navigationBarColor = '#003a66'
+const navBarBackgroundColor = '#003a66'
+const statusBarColor = '#002b4c'
+const tabFontFamily = 'BioRhyme-Bold'
+
+export const animationType = 'fade'
+
+export const tabsStyle = {
+  tabBarBackgroundColor,
+  navBarButtonColor,
+  tabBarButtonColor,
+  navBarTextColor,
+  tabBarSelectedButtonColor,
+  navigationBarColor,
+  navBarBackgroundColor,
+  statusBarColor,
+  tabFontFamily
+}
+
+export const appStyle = {
+  tabBarBackgroundColor,
+  navBarButtonColor,
+  tabBarButtonColor,
+  navBarTextColor,
+  tabBarSelectedButtonColor,
+  navigationBarColor,
+  navBarBackgroundColor,
+  statusBarColor,
+  tabFontFamily
+}
+
+export const tabs = [
+  {
+    label: 'SignIn',
+    screen: 'example.Types',
+    icon: require('../Assets/Images/swap.png'),
+    title: 'Sign In Screen'
+  }, {
+    label: 'SignIn',
+    screen: 'example.Types',
+    icon: require('../Assets/Images/swap.png'),
+    title: 'Sign In Screen'
+  }
```

Comment:
Next PR will solve this,
![image](https://user-images.githubusercontent.com/823150/27771160-73c94746-5f1f-11e7-9459-e40651464b85.png)
![image](https://user-images.githubusercontent.com/823150/27771162-7b5e76d4-5f1f-11e7-99d2-fbf844d5be83.png)
![image](https://user-images.githubusercontent.com/823150/27771163-8652222a-5f1f-11e7-845f-3bde14db0684.png)